support-files/mysql.spec.sh

```diff
@@ -124,104 +120,42 @@
 %endif
 
 # ----------------------------------------------------------------------------
-# Packager
+# Distribution support
 # ----------------------------------------------------------------------------
-%if %{undefined mysql_packager}
-%define mysql_packager MySQL Build Team <build@mysql.com>
+
+# Disable post build checks for time being.
+BuildConflicts: post-build-checks rpmlint
+
+BuildRequires: gcc-c++ ncurses-devel perl zlib-devel cmake libaio-devel bison flex
+
+%if 0%{?rhel} == 6 || 0%{?rhel} == 7 || 0%{?fedora} == 20 || 0%{?fedora} == 21
+BuildRequires: time
 %endif
 
-# ----------------------------------------------------------------------------
-# Distribution support
-# ----------------------------------------------------------------------------
-%if %{undefined distro_specific}
-%define distro_specific 0
+%if 0%{?suse_version}
+%if 0%{?suse_version} == 1110
+BuildRequires: gdbm-devel gperf openldap2-client procps pwdutils
 %endif
-%if %{distro_specific}
```

Comment:
distro_buildreq and distro_requires are handled explicitly in appropriate places. distro_releasetag will be replaced with dist macro provided by platform (or defined explicitly in case of suse). distro_description was used only in Distribution field which is optional, so I decided to drop it for simplicity.